deploy/deploy_test.go

```diff
@@ -186,7 +255,12 @@ func TestKindSpec(t *testing.T) {
 				"docker": "local",
 			},
 		},
-		execer:  exec.NewFakeExecer(nil, errors.New("unable to pull")),
+		resp: []fexec.Response{
+			{Cmd: "kind", Args: []string{"create", "cluster", "--name", "test"}},
+			{Cmd: "docker", Args: []string{"pull", "docker"}},
```

Comment:
i think the err should be here during the pull

Reply:
Done